shared-context/e2e/checkSharedContextApps.cy.ts

```diff
@@ -0,0 +1,50 @@
+import {BaseMethods} from "../../cypress/common/base";
+import {baseSelectors} from "../../cypress/common/selectors";
+import {Constants} from "../../cypress/fixtures/constants";
+const basePage: BaseMethods = new BaseMethods()
+
+const appsData = [
+    {
+        header: Constants.elementsText.sharedContext.header,
+        subheader: Constants.elementsText.sharedContext.app1.subheader,
+        paragraph: Constants.elementsText.sharedContext.app1.paragraph,
+        host: 3001
+    },
+
+    {
+        header: Constants.elementsText.sharedContext.header,
+        subheader: Constants.elementsText.sharedContext.app2.subheader,
+        paragraph: Constants.elementsText.sharedContext.app2.paragraph,
+        host: 3002
+    }
+]
+
+appsData.forEach((
+    property: {
+        header: string,
+        subheader: string,
+        paragraph: string,
+        host: number
+    }
+) => {
+    describe(`Check app1 and app2`, () => {
+        beforeEach(() => {
+            basePage.openLocalhost(property.host)
+        })
+        it (`Check ${property.subheader} starts and running`, () => {
```

Comment:
it (`Check ${property.subheader} elements (header, subHeader, paragraph)`, () => {